docs/guide/typescript/theming.md

```diff
@@ -132,3 +132,50 @@ App.start({
 You could also transpile scss into css using a bundler
 and simply passing the path of the resulting css file to `css`.
 :::
+
+## Autoreloading SCSS for Improved Developer Experience
+
+To enhance the development workflow, you can add the following
+code to your app.ts file. This will automatically recompile
+your style.scss file and reload it in the app whenever
+changes are saved. With this setup, the SCSS will be recompiled
+and the app will be updated instantly on each save.
+
+Use the following code to get started:
+
+:::code-group
+
+```ts [app.ts]
+const scss = `./style.scss` // Path to you style file
```

Comment:
don't use template literals unless required (backticks)

Reply:
```ts
const scss = `${SRC}/style.scss` 
```

`SRC` could be used

Reply:
Is `${SRC}` the new built-in equivalent of `${App.configDir}` or is it just something the user has to set? Because I couldn't find this anywhere in the docs and thus just used `./`.

Reply:
> AGS also defines a global `SRC` variable which will be replaced by the value of the --src flag. By default it will point to the directory of `entryfile`.

Source: [Bundling projects(ags docs)](https://aylur.github.io/ags/guide/bundling.html#bundling-projects)
> Since Astal is a library it lost App.configDir.
[...]
In cases where you still have to refer to the source dir, AGS defines a global SRC variable.

Source: [release notes for ags v2](https://github.com/Aylur/ags/releases/tag/v2.0.0#:~:text=In%20cases%20where%20you%20still%20have%20to%20refer%20to%20the%20source%20dir%2C%20AGS%20defines%20a%20global%20SRC%20variable.)
Also (kind of) relevant:  [setting up a project(ags docs)](https://aylur.github.io/ags/guide/init.html#setting-up-a-project:~:text=as%20well%20as%20that%20a%20global%20SRC%20variable%20is%20available)

Reply:
Astal docs should not use AGS features as it is meant to be independent. It is a library, there is no concept of a src directory, it's the responsibility of build tools to handle that. `./` is fine for the example, it should just have a note under it like the [one here](https://aylur.github.io/astal/guide/typescript/theming#from-file-at-startup)